packages-ts/gauntlet-terra-contracts/src/lib/contracts.ts

```diff
@@ -31,19 +31,19 @@ export type Contract = {
 
 export type Contracts = Record<CONTRACT_LIST, Contract>
 
-export const loadContracts = (version): Contracts => {
-  return Object.values(CONTRACT_LIST).reduce((agg, id) => {
+export const loadContracts = async (version): Promise<Contracts> => {
```

Comment:
We can remove this function, don't think we use it anymore

Reply:
Good catch! just fixed this